### PR TITLE
NO-ISSUE: improve cleanup script

### PR DESCRIPTION
**Pull request by eifrach**

### CODE REVIEW

File: src/cleanup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import pathlib
+import shutil
+
+REMOVE_FILES = ["*.py[co]", "test_infra.log"]
+REMOVE_FOLDERS = ["__pycache__", "UNKNOWN.egg-info", "build", "assisted-service", "reports", ".mypy_cache"]
```
Comment (danmanor) [inline src/cleanup.py:5]: I don't think we should remove assisted-service directory, we will have to clone it every time

Comment (eifrach) [reply src/cleanup.py:5]: when you do clean, don't you think we should clean everything?

Comment (danmanor) [reply src/cleanup.py:5]: We have logic that sets that just resets to master if the repository already exists, so I think i is redundant. I think whenever we can we should avoid cloning

Comment (eifrach) [reply src/cleanup.py]: changed

**Review by danmanor — APPROVED** [review]

**Pull request merged**